scipy/optimize/_qap.py

```diff
@@ -0,0 +1,413 @@
+import numpy as np
+import math
+from . import linear_sum_assignment, minimize_scalar, OptimizeResult
+
+
+def quadratic_assignment(
+    cost_matrix,
+    dist_matrix,
+    seed=np.array([[], []]).T,
+    maximize=False,
+    n_init=1,
+    init_method="barycenter",
+    max_iter=30,
+    shuffle_input=True,
+    eps=0.05,
+):
+    """
+    Solve the quadratic assignment problem
+
+    This class solves the Quadratic Assignment Problem and the Graph
+    Matching Problem (QAP) through an implementation of the Fast
+    Approximate QAP Algorithm (FAQ) (these two problems are the same up
+    to a sign change) [1]_.
+
+    This algorithm can be thought of as finding an alignment of the
+    vertices of two graphs which minimizes the number of induced edge
+    disagreements, or, in the case of weighted graphs, the sum of squared
+    differences of edge weight disagreements. The option to add seeds
+    (known vertex correspondence between some nodes) is also available
+    [2]_.
+
+
+    Parameters
+    ----------
+    cost_matrix : 2d-array, square
+        A square adjacency matrix
+
+    dist_matrix : 2d-array, square
+        A square adjacency matrix
+
+    seed : 2d-array, optional, (default = np.array([[], []]).T)
+        Allows the user apply a seed, fixing part of the matching between
+        the two adjacency matrices.
+        For column 1, each entry is an index of a node in 'cost_matrix'.
+        For column 2, each entry is an index of a node in 'dist_matrix'.
+        The elements of 'seed[:, 0]' and 'seed_dist[:,1]' are vertices which
```

Comment:
```suggestion
        The elements of ``seed[:, 0]`` and ``seed_dist[:, 1]`` are vertices which
```
I think you saw my previous comment about backticks ( ` ), but it looks like there some confusion with single quotes ( ' ).  Please see [A Guide to NumPy/SciPy Documentation](https://numpy.org/devdocs/docs/howto_document.html#sections) and search for "backticks".